react-native/debugging/analyse-bug.mo.md

```diff
@@ -0,0 +1,185 @@
+# [MO] How to analyse and react to a bug on a React-Native project ?
+
+## Owner: [Thomas Pucci](https://github.com/tpucci)
+
+## Prerequisites
+
+* 🐛 You have a bug, obviously.
+* ⚛️ It lives in a React-Native project.
+
+## Scrum preamble
+
+You just spotted a bug. What should you do ?
+
+{% hint style='info' %}
+
+For M33 People:
+
+First thing first: communicate with your Product Owner and follow this [mode of operation](http://blog.m33.network/2017/07/i-just-detected-a-bug-what-should-i-do/).
+
+{% endhint %}
+
+### Option 1: You know where it comes from and how to fix it
```

Comment:
You never know from first sight where it comes from, how much spend time should you spend before switching to option 2?